src/main/java/io/vertx/redis/RedisService.java

```diff
@@ -0,0 +1,1634 @@
+   *
+   * @param args    JsonArray [{"name":"key","type":"key","multiple":true},{"name":"timeout","type":"integer"}]
+   * @param handler Handler for the result of this call.
+   * @since 2.0.0
+   * group: list
+   */
+  void brpop(JsonArray args, Handler<AsyncResult<JsonArray>> handler);
+
+  /**
+   * Pop a value from a list, push it to another list and return it; or block until one is available
+   *
+   * @param args    JsonArray [{"name":"source","type":"key"},{"name":"destination","type":"key"},{"name":"timeout","type":"integer"}]
+   * @param handler Handler for the result of this call.
+   * @since 2.2.0
+   * group: list
+   */
+  void brpoplpush(JsonArray args, Handler<AsyncResult<Void>> handler);
+
+  /**
+   * Kill the connection of a client
+   *
+   * @param args    JsonArray [{"name":"ip:port","type":"string","optional":true},{"command":"ID","name":"client-id","type":"integer","optional":true},{"command":"TYPE","type":"enum","enum":["normal","slave","pubsub"],"optional":true},{"command":"ADDR","name":"ip:port","type":"string","optional":true},{"command":"SKIPME","name":"yes/no","type":"string","optional":true}]
+   * @param handler Handler for the result of this call.
+   * @since 2.4.0
+   * group: server
+   */
+  void clientKill(JsonArray args, Handler<AsyncResult<Long>> handler);
+
+  /**
+   * Get the list of client connections
+   *
+   * @since 2.4.0
+   * group: server
+   */
+  void clientList(Handler<AsyncResult<String>> handler);
+
+  /**
+   * Get the current connection name
+   *
+   * @since 2.6.9
+   * group: server
+   */
+  void clientGetname(Handler<AsyncResult<String>> handler);
+
+  /**
+   * Stop processing commands from clients for some time
+   *
+   * @param args    JsonArray [{"name":"timeout","type":"integer"}]
+   * @param handler Handler for the result of this call.
+   * @since 2.9.50
+   * group: server
+   */
+  void clientPause(JsonArray args, Handler<AsyncResult<String>> handler);
+
+  /**
+   * Set the current connection name
+   *
+   * @param args    JsonArray [{"name":"connection-name","type":"string"}]
+   * @param handler Handler for the result of this call.
+   * @since 2.6.9
+   * group: server
+   */
+  void clientSetname(JsonArray args, Handler<AsyncResult<String>> handler);
+
+  /**
+   * Get array of Cluster slot to node mappings
+   *
+   * @since 3.0.0
+   * group: server
+   */
+  void clusterSlots(Handler<AsyncResult<JsonArray>> handler);
+
+  /**
+   * Get array of Redis command details
+   *
+   * @since 2.8.13
+   * group: server
+   */
+  void command(Handler<AsyncResult<JsonArray>> handler);
+
+  /**
+   * Get total number of Redis commands
+   *
+   * @since 2.8.13
+   * group: server
+   */
+  void commandCount(Handler<AsyncResult<Long>> handler);
+
+  /**
+   * Extract keys given a full Redis command
+   *
+   * @since 2.8.13
+   * group: server
+   */
+  void commandGetkeys(Handler<AsyncResult<JsonArray>> handler);
```

Comment:
No. There are a few of these in Redis that simple are commands where no input is required and simply return a result.